packages/gatsby-theme/src/components/Image/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/** @jsx jsx */
+import { Box, jsx } from 'theme-ui'
+import Img from 'gatsby-image'
+import RichText from '../RichText'
+
+const Image = React.forwardRef(({ variant, caption, image, ...props }, ref) => {
+  const isGatsbyImage = !image.src
+  return (
+    <Box as="figure" __css={{ m: 0 }} {...props}>
+      {isGatsbyImage ? (
+        <Img ref={ref} variant={variant} __themeKey="images" {...image} />
```

Comment:
I think you can check with Carlos if the variant is necessary in your <Img /> tag because I remember he deleted it.

Reply:
You were right. I checked and the {variant} isn't necessary on <Img>